model/config.go

```diff
@@ -1927,15 +1938,17 @@ type SamlSettings struct {
 	PrivateKeyFile        *string
 
 	// User Mapping
-	IdAttribute        *string
-	GuestAttribute     *string
-	FirstNameAttribute *string
-	LastNameAttribute  *string
-	EmailAttribute     *string
-	UsernameAttribute  *string
-	NicknameAttribute  *string
-	LocaleAttribute    *string
-	PositionAttribute  *string
+	IdAttribute          *string
+	GuestAttribute       *string
+	EnableAdminAttribute *bool
```

Comment:
Is there any reason to need this `EnableAdminAttribute` setting here, when we don't have an equivalent one for `GuestAttribute`? Makes sense to default the `AdminAttribute` to blank and that means "disabled" and if there is any value, is enabled? Is not that behavior already the expected one?

Reply:
@jespino 
Actually we do have one for GuestAttribute and GuestFilter. It's the `Enable` in the `GuestAccountSettings` in config.json. However, that has more of an impact than just turning the `GuestAttribute` and `GuestFilter` on and off.

Without the `EnableAdminAttribute` and `EnableAdminFilter`, the problem we run into is if the filter/attribute is blank, then all members who are admins will become members when they login even if we don't want that to happen.  If we disable the filter, then the admins will not be demoted on the next login. This was the reason we added the Enable.